doc/devel/color_changes.rst

```diff
@@ -0,0 +1,133 @@
+.. _color_changes:
+
+*********************
+Default Color changes
+*********************
+
+As discussed at length elsewhere [insert links], ``jet`` is an
+empirically bad color map and should not be the default color map.
+Due to the position that changing the appearance of plot is a back
```

Comment:
maybe just say "Due to the position that changing the appearance of the plot breaks backward compatibility, ..."